onyxia-api/src/main/resources/multiverse.json

```diff
@@ -15,6 +15,14 @@
       "maintainer": "innovation@insee.fr",
       "location": "https://inseefrlab.github.io/Universe-Datascience/universe.json",
       "status": "PROD"
+    },
+    {
+      "id": "inseefrlab-catalog-wip",
+      "name": "Datascience catalog WIP",
+      "description": "WIP for services for datascientists. https://github.com/InseeFrLab/Universe-Datascience branch Test",
+      "maintainer": "innovation@insee.fr",
+      "location": "https://inseefrlab.github.io/Universe-Datascience/universe-test.json",
+      "status": "INTERNAL"
```

Comment:
The 3 status currently defined are : PROD (should be displayed and used by everyone), TEST (should not be displayed on UI, we may add a "enable beta features" on the UI oneday to activate it) and INTERNAL (should only be used by the API - e.g : the cloudshell).  
I suggest we change the status of this dev catalog from INTERNAL to TEST.  
Sorry, we really need to explicit & document those kind of things.

Reply:
We change the status from INTERNAL to TEST